GPSTest/src/main/java/com/android/gpstest/view/GpsSkyView.java

```diff
@@ -73,15 +73,20 @@
 
     private boolean mStarted;
 
-    private float mSnrCn0s[], mElevs[], mAzims[];  // Holds either SNR or C/N0 - see #65
+    private float[] mSnrCn0s;
+    private float[] mElevs;
+    private float[] mAzims;  // Holds either SNR or C/N0 - see #65
```

Comment:
Could you move this comment up to the line with `mSnrCn0s`?

Reply:
Hey @barbeau, happy to make the change.. It's now done.. 

Also, thanks to you for the prompt eyes on this!